public/src/components/channelManagement/bannerTests/bannerTestVariantEditor.tsx

```diff
@@ -280,7 +280,10 @@ const BannerTestVariantContentEditor: React.FC<BannerTestVariantContentEditorPro
 
           {(template === BannerTemplate.ContributionsBanner ||
             template === BannerTemplate.InvestigationsMomentBanner ||
-            template === BannerTemplate.ElectionAuMomentBanner) && (
+            template === BannerTemplate.ElectionAuMomentBanner ||
+            template === BannerTemplate.PostElectionAuMomentAlbaneseBanner ||
+            template === BannerTemplate.PostElectionAuMomentHungBanner ||
+            template === BannerTemplate.PostElectionAuMomentMorrisonBanner) && (
```

Comment:
I wonder if it's worth thinking of an easier way to do this? I think it's fine for now, given it's just a single field that is being conditionally rendered